includes/definitions/discovery/junos.yaml

```diff
@@ -4,14 +4,23 @@ modules:
         data:
             -
                 oid: JUNIPER-MIB::jnxOperatingTable
-                total: JUNIPER-MIB::jnxOperatingDRAMSize
+                total: JUNIPER-MIB::jnxOperatingMemory
                 percent_used: JUNIPER-MIB::jnxOperatingBuffer
                 descr: '{{ JUNIPER-MIB::jnxOperatingDescr }}'
+                precision: 1048576
                 skip_values:
                     -
-                        oid: JUNIPER-MIB::jnxOperatingDRAMSize
+                        oid: JUNIPER-MIB::jnxOperatingMemory
                         op: '='
                         value: 0
+                    -
+                        oid: JUNIPER-MIB::jnxOperatingDescr
+                        op: 'contains'
+                        value: 'Fan'
+                    -
+                        oid: JUNIPER-MIB::jnxOperatingDescr
+                        op: 'contains'
+                        value: 'Sensor'
```

Comment:
For some reason, on MX-80 it still discovers a "TFEB Intake Temperature Sensor" after applying this patch. Memory units are in GiB now.

Reply:
This "TFEB Intake Temperature Sensor" is discovered in CPU and Memory sections.

Debug capture shows this line on mempools: jnxOperatingDescr.* = "TFEB Intake temperature sensor"

Reply:
Also adding "TFEB" into the CPU and Memory skip values seems to achieve the desired outcome atleast with my devices.

Reply:
Looks like 'sensor' is lower cased in your example - I can add lowercase versions of each.